portal-api/src/createApp.js

```diff
@@ -32,6 +32,15 @@ app.use('/v1', openRouter);
 app.use('/v1', authRouterAllowXss);
 app.use('/v1', authRouter);
 
+// Return 200 on get to / to confirm to Azure that
+// the container has started successfully:
+const rootRouter = express.Router();
+rootRouter.get('/', async (req, res) => {
+  res.status(200).send();
+});
```

Comment:
You don't need the `async` here because you don't have `await`, unless you want to force the anonymous function to return a promise for some reason